srcs/utility.cpp

```diff
@@ -0,0 +1,26 @@
+#include <utility.hpp>
```

Comment:
spliting can also be done with` std::getline` like [this](https://stackoverflow.com/questions/5167625/splitting-a-c-stdstring-using-tokens-e-g)

Reply:
I tried that before. The downside is that only works for delimters that are one character, but I also want to be able to split on CRLF which is two characters ("\r\n").

But I really appricate that you took time to look at the code :)